src/Components/ExpenseBlock/ExpenseQuestion.js

```diff
@@ -96,7 +96,15 @@ const ExpenseQuestion = ({ expenseData, allExpensesData, setAllExpenses, deleteE
   };
 
   const getExpenseSourceLabel = (expenseSourceName) => {
-    return expenseOptions[expenseSourceName];
+    return expenseSourceName ? (
+      <>
+        {'('}
+        {expenseOptions[expenseSourceName]}
+        {')?'}
+      </>
+    ) : (
+      '?'
+    );
```

Comment:
Guard clauses are usually easier to read than ternaries. Also I refactored the expense option part to just return a string instead of JSX.
```suggestion
    if (expenseSourceName) {
      return `(${expenseOptions[expenseSourceName]})?`;
    }
    
    return '?';
```

Reply:
Thanks for the suggestion. Currently if I just return as string directly, like this ``return `(${expenseOptions[expenseSourceName]})?`;``, I got [object Object]. Let me know if I need to temporarily return as JSX.

Reply:
I did not realize that `expenseOptions[expenseSourceName]` returned a `FormattedMessage`. Here is an updated suggestion.
```suggestion
    if (expenseSourceName) {
      return <>({expenseOptions[expenseSourceName]})?</>;
    }
    
    return '?';
```